### PR TITLE
Enhancement: use ids instead of names and codes.

### PR DESCRIPTION
**Pull request by devgenix**
The following have been worked on:

- [x] Use ids to get resources instead of codes or names as they are unreliable. 
- [x] Uninstalled Channels, it's not needed now. 
- [x] Set development to use django internal cache instead of using Redis in production

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -10,28 +10,68 @@ services:
         build:
             context: .
             dockerfile: ./Dockerfile
-        command: sh -c "python manage.py runserver"
-        # volumes:
-        #     - .:/univast
+        command: sh -c "python manage.py migrate --noinput && celery -A Univast worker -l info & gunicorn --bind=0.0.0.0:9000 --timeout 600 Univast.wsgi --reload"
```
Comment (aybruhm) [inline docker-compose.yaml]: There's no need to include the start celery worker command anymore. It has it's own service.

File: docker-compose.yaml [hunk 2]
```diff
@@ -10,28 +10,68 @@ services:
...
+        depends_on:
+        # - db
+        # - redis
+        - web
+
+    flower:
+        container_name: "Univast_flower"
+        build:
+            context: .
+        command: celery -A Univast --broker=${CELERY_BROKER_URL} flower --port=5557
```
Comment (aybruhm) [inline docker-compose.yaml]: If someone were to navigate to ourwebsite.com:5557, they would have full access to the analytics dashboard. Could you please add [authentication](https://flower.readthedocs.io/en/latest/auth.html) to secure it?

**Review by aybruhm — CHANGES_REQUESTED** [review]

**Comment by devgenix** [comment]
@aybruhm resolved requested changes. 

Note that docker is not used to deploy on production.

**Review by aybruhm — APPROVED** [review]
Awesome job, man.

**Comment by aybruhm** [comment]
> @aybruhm resolved requested changes. 
> 
> Note that docker is not used to deploy on production. 

Oh, noted.

**Pull request merged**